.github/workflows/sync.yml

```diff
@@ -0,0 +1,52 @@
+name: Sync
```

Comment:
I would call this Deploy. `sync` is just an implementation detail of the `awscli`. I thought you were referring to the Github PR synchronize event at first.